src/software/backend/input/network/networking/network_client.cpp

```diff
@@ -192,10 +192,13 @@ void NetworkClient::filterAndPublishVisionData(SSL_WrapperPacket packet)
 
 void NetworkClient::filterAndPublishGameControllerData(Referee packet)
 {
-    RefboxGameState game_state = network_filter.getRefboxGameState(packet);
-    world.updateRefboxGameState(game_state);
+    if (world.field().isFieldValid())
+    {
+        RefboxGameState game_state = network_filter.getRefboxGameState(packet);
+        world.updateRefboxGameState(game_state);
```

Comment:
These lines updating the world can be outside the if-block. We only need to prevent the world from publishing. It's still useful to update state internally in the meantime.

Reply:
done